src/main/java/org/terasology/climateConditions/HypothermiaSystem.java

```diff
@@ -16,31 +16,47 @@
 package org.terasology.climateConditions;
 
 import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.entity.lifecycleEvents.OnAddedComponent;
 import org.terasology.entitySystem.event.ReceiveEvent;
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
 import org.terasology.logic.characters.CharacterMovementComponent;
-import org.terasology.logic.location.LocationComponent;
+import org.terasology.logic.delay.DelayManager;
 import org.terasology.logic.players.PlayerCharacterComponent;
 import org.terasology.physics.events.MovedEvent;
+import org.terasology.registry.In;
+
+import static org.terasology.climateConditions.VisibleBreathingSystem.VISIBLE_BREATH_ACTION_ID;
 
 @RegisterSystem(value = RegisterMode.AUTHORITY)
 public class HypothermiaSystem extends BaseComponentSystem {
+
+    @In
+    private DelayManager delayManager;
+
     private float thresholdHeight = 60f;
+    private int initialDelay = 5000;
+    private int breathInterval = 7000;
 
     @ReceiveEvent(components = {PlayerCharacterComponent.class, CharacterMovementComponent.class})
-    public void observeDangerZone(MovedEvent event, EntityRef player, LocationComponent location, CharacterMovementComponent movement) {
+    public void observeDangerZone(MovedEvent event, EntityRef player) {
         //TODO: react on OnBiomeChangedEvent to handle the danger zone
         float height = event.getPosition().getY();
         float lastHeight = height - event.getDelta().getY();
         if (height > thresholdHeight && lastHeight <= thresholdHeight) {
-                player.addOrSaveComponent(new HypothermiaComponent());
+            player.addOrSaveComponent(new HypothermiaComponent());
         }
         if (height < thresholdHeight && lastHeight >= thresholdHeight) {
-            if(player.hasComponent(HypothermiaComponent.class)){
+            if (player.hasComponent(HypothermiaComponent.class)) {
                 player.removeComponent(HypothermiaComponent.class);
+                delayManager.cancelPeriodicAction(player, VISIBLE_BREATH_ACTION_ID);
             }
         }
     }
+
+    @ReceiveEvent(components = {HypothermiaComponent.class})
+    public void onHypothermia(OnAddedComponent event, EntityRef player, HypothermiaComponent hypothermia) {
+        delayManager.addPeriodicAction(player, VISIBLE_BREATH_ACTION_ID, initialDelay, breathInterval);
```

Comment:
I think for clarity I'd not do a static import of `VISIBLE_BREATH_ACTION_ID` but only import `VisibleBreathingSystem` and reference this as
```suggestion
        delayManager.addPeriodicAction(player, VisibleBreathingSystem.VISIBLE_BREATH_ACTION_ID, initialDelay, breathInterval);
```

This makes it clearer where this constant is coming from.